sources/packages/web/src/components/aest/OfferingChangeRequest.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+  <offering-form :data="initialData"></offering-form>
+</template>
+
+<script lang="ts">
+import { onMounted, ref, SetupContext } from "vue";
+import { EducationProgramOfferingService } from "@/services/EducationProgramOfferingService";
+import { EducationProgramService } from "@/services/EducationProgramService";
+import { OfferingFormBaseModel, OfferingStatus } from "@/types";
+import { AESTRoutesConst } from "@/constants/routes/RouteConstants";
+import { BannerTypes } from "@/components/generic/Banner.models";
+import OfferingForm from "@/components/common/OfferingForm.vue";
+
+export default {
+  components: {
+    OfferingForm,
+  },
+  props: {
+    programId: {
+      type: Number,
+      required: true,
+    },
+    offeringId: {
+      type: Number,
+      required: true,
+    },
+  },
+  emits: ["getHeaderDetails"],
```

Comment:
It does not seem to be the right way to use an event. I suggest just calling the API again to grab the data of something similar on `institution\ViewOfferingChangeRequest.vue`.